public/guides/upgrade.md

```diff
@@ -1,39 +1,241 @@
+
+
+### Other changes
+
+- New apps will default to using `HTTP` APIs, but can be configured as `REST` APIs – more on that below
+
+
+### Removed
+
+- Removed experimental support for `@http` static mocks
+  - This was a very obscure experimental feature and should not impact anyone (but can be restored by building a macro)
+
+
+### <span id=architect-7-breaking-changes>Breaking changes
+
+All breaking changes in Architect 7 pertain to local development and testing with Sandbox, Architect's development environment. **But good news: are backward compatibility paths for Architect 6 that only require a few seconds of your time.** Read on!
+
+A core goal of Sandbox is to operate entirely locally and offline; that means no phoning home to AWS to introspect your current application configuration. Sandbox must run entirely from your machine using only your Architect project manifest. Thus, with the addition of `HTTP` APIs, Sandbox must now have a default API type.
+
+It naturally follows that Sandbox's default would be `HTTP` (with Lambda payload format version 2.0), since that is now the default for new projects. Because API Gateway `HTTP` APIs introduced breaking changes with `REST` APIs, existing projects may not function correctly until reconfigured to enter `REST` mode.
+
+
+#### Configuring the API type in Sandbox
+
+The following are the valid settings for API types:
+- **`http` (default)** - `HTTP` API + Lambda payload format version 2.0
+  - `httpv2` (aliased to `http`)
+- **`httpv1`** - `HTTP` API + Lambda payload format version 1.0; use this if you'd like to take advantage of `HTTP` APIs, but aren't yet ready to refactor your code for payload format 2.0
+- **`rest` (previous default)** - `REST` API + original API Gateway payload format (essentially the same as what is now called Lambda payload format version 1.0)
+
+You can safely toggle between `http/httpv2` and `httpv1` without destroying your current API URLs (although code may no longer function; see `@architect/functions` below for information about a seamless upgrade).
+
+Configure your project's API type to `REST` (for example) one of the following ways:
+
+1. Architect project manifest
+```arc
+# app.arc|.arc|arc.yaml|etc.
+@aws
+apigateway rest
+```
+
+or:
+
+2. Environment variable
+```sh
+ARC_API_TYPE=rest npx arc sandbox
+```
+
+
+#### Sandbox test environment changes
+
+- Several seldom used and largely undocumented Sandbox module APIs have breaking changes:
+  - `sandbox.start()` no longer returns a function to shut down, and should now be shut down directly with `sandbox.end()`
+  - `sandbox.db()` is now `sandbox.tables()`
+  - `http.close()` is now `http.end()`
+  - `events.start()` & `tables.start()` no longer return server objects to be invoked with `.close()`, and should now be shut down directly with `events.end()` and `tables.end()`
+- tldr the Sandbox module API is now:
```

Comment:
I suggest we replace this bullet point as well as its sub-points and link to sandbox's readme, perhaps something like:

```suggestion
For Sandbox API details, check out [the Sandbox README](https://github.com/architect/sandbox#api).
```

Reply:
Got it!